bundle/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -1258,6 +1259,10 @@ spec:
                   value: quay.io/netobserv/network-observability-console-plugin:v1.9.1-community
                 - name: RELATED_IMAGE_CONSOLE_PLUGIN_COMPAT
                   value: quay.io/netobserv/network-observability-console-plugin-pf4:v1.8.2-community
+                - name: NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
@jpinsonneau - can we append this env var instead of adding it in middle so that index # for the existing vars doesn't change? We have several scripts and CI steps where we patch  csv based which happens based on index of env var.

Reply:
Sure, I cant put it in the end of the list if it help :smile_cat: